src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -138,29 +133,19 @@ async function getAdditionalFacilityInfo(futureAppointments) {
   // Get facility ids from non-VA appts or requests
   const nonVaFacilityAppointmentIds = futureAppointments
     .filter(
-      appt =>
-        !isVideoAppointment(appt) && (appt.vaos?.isCommunityCare || !appt.vaos),
+      appt => !appt.vaos?.isVideo && (appt.vaos?.isCommunityCare || !appt.vaos),
     )
     .map(appt => appt.facilityId || appt.facility?.facilityCode);
 
-  const videoAppointmentIds = futureAppointments
```

Comment:
No need for a separate loop here, we can get the facility id for all VA appointments using the helper